### ISSUE TITLE
Service worker files pile up?

### ISSUE THREAD

**Issue by tehKaiN** [BartmanAbyss/vscode-amiga-debug#204]
I've just inspected my disk and have something like 30GB of service worker cache. Looks like a lot of 16MB files.

![obraz](https://user-images.githubusercontent.com/16180344/213555273-75924662-47ba-4754-a7c9-ef0b9f94bb3f.png)

When looking at one of them I see:

![obraz](https://user-images.githubusercontent.com/16180344/213555427-0f79ec8d-0bdd-426d-846c-88b636d223e9.png)

looks like some kind of cache for .amigaprofile files? How to nicely get rid of them? Does the extension lack some kind of cleanup in its code?

**Comment by BartmanAbyss** [comment]
Wow, these aren't mine though. 
I store the temporary .amigaprofile files in %temp%, and you can clean those via this command:
```json
				"category": "Amiga",
				"command": "amiga.cleanTemp",
				"title": "Clean Temp Files"
```
However, if you have any idea on what that `Service Worker Cache` is and how to clean it up, please let me know.

**Comment by tehKaiN** [comment]
yeah I know that, but I guess vscode makes a copy of them for some strange reason and in some strange format. Perhaps it's some kind of caching for displaying profiler files?

I'll try to investigate it by removing those manually and trying to get them to generate again.

Do you have similar files created or is it just me?

**Comment by BartmanAbyss** [comment]
My CacheStorage folder here is 4.4GB. I suppose it's something like a browser cache folder, as it seems to contain caches of files requested by extensions etc. e.g. `https://file+.vscode-resource.vscode-cdn.net/c%3A/Users/barto/Documents/Visual_Studio_Code/amiga-debug/dist/client.js`
